ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -377,10 +363,166 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
           {filters}
         </small>
       )}
+    </div>
+  )
+}
+
+GenotypeQuality.propTypes = {
+  genotype: PropTypes.object,
+  variant: PropTypes.object,
+}
+
+const LegacyAdditionalSampleTypePopup = ({ genotype, variant, isHemiX, genesById }) => {
+  const hasConflictingNumAlt = genotype.otherSample && genotype.otherSample.numAlt !== genotype.numAlt
+
+  return (
+    <div>
+      {genotype.otherSample && (
+      <Popup
+        header="Additional Sample Type"
+        trigger={<Icon name="plus circle" color={hasConflictingNumAlt ? 'red' : 'green'} />}
+        content={
+          <div>
+            {hasConflictingNumAlt && (
+              <div>
+                <VerticalSpacer height={5} />
+                <Alleles genotype={genotype.otherSample} variant={variant} isHemiX={isHemiX} />
+                <VerticalSpacer height={5} />
+              </div>
+            )}
+            {genotypeDetails(genotype.otherSample, variant, genesById)}
+          </div>
+        }
+      />
+      )}
+    </div>
+  )
+}
+
+LegacyAdditionalSampleTypePopup.propTypes = {
+  genotype: PropTypes.object,
+  variant: PropTypes.object,
+  isHemiX: PropTypes.bool,
+  genesById: PropTypes.object,
+}
+
+export const MultiSampleTypeAlleles = React.memo(({ genotypes, variant, individual, isHemiX, isCompoundHet }) => {
+  const warnings = genotypes.reduce((acc, genotype) => {
+    const warning = getWarningsForGenotype(genotype, variant, individual, isHemiX, isCompoundHet)
+    if (warning) {
+      acc[genotype.sampleType] = warning
+    }
+    return acc
+  }, {})
+
+  const hasDifferentNumAlt = genotypes.some(genotype => genotype.numAlt !== genotypes[0].numAlt)
+
+  return (
+    <Alleles
+      genotype={genotypes[0]}
+      variant={variant}
+      isHemiX={isHemiX}
+      warning={(Object.keys(warnings).length > 0 || hasDifferentNumAlt) && (
+        <div>
+          {Object.entries(warnings).map(([sampleType, warning]) => (
+            <div key={sampleType}>
+              <b>
+                {sampleType}
+                Warning:
+              </b>
+              {warning}
+            </div>
+          ))}
+          {hasDifferentNumAlt && (
+            <span>
```

Comment:
This is going to look weird if you have a sample-type specific warnings and also different number of genotypes - it will be something like 
```
WES: Potential UPD/ Hemizygosity
WGS: Potential UPD/ Hemizygosity. Variant absent in parents
Genotypes differ across sample types
WES:
  G/C
WGS:
  C/C
```

I think the differing genotypes warning belongs at the top, and also that rather than combining all the warnings per sample type, we should show each warning on each own row, with an indicator if its only foor some sample types:

```
Genotypes differ across sample types
WES:
  G/C
WGS:
  C/C
Potential UPD/ Hemizygosity
Variant absent in parents (WGS)
```

Reply:
<img width="282" alt="Screenshot 2024-10-23 at 4 22 43 PM" src="https://github.com/user-attachments/assets/bb1bb02b-0bae-492b-8ff5-ee0707191440">

I agree here!